x86/SearchMacros.asm

```diff
@@ -442,7 +451,9 @@ Display	2, "Search(alpha=%i1, beta=%i2,	depth=%i8, cutNode=%i9)	called%n"
 		lea   edx, [rcx+1]
 	      movzx   r9d, byte[.cutNode]
 		not   r9d		; not used in qsearch case
-	       call   r12
+
```

Comment:
Temporarily added in pop r12 and push r12 to lines 454 and 456, respectively to ensure I wasn't hammering r12. The bench didn't change, so the spots where I use r12 should be valid.